pkg/ddb/builder_putitem.go

```diff
@@ -48,7 +50,7 @@ func (b *putItemBuilder) ReturnAllOld() PutItemBuilder {
 
 func (b *putItemBuilder) Build(item interface{}) (*dynamodb.PutItemInput, error) {
 	if b.returnType != nil && *b.returnType != dynamodb.ReturnValueNone && !isPointer(item) {
-		return nil, fmt.Errorf("the provided old value has to be a pointer")
+		return nil, errors.New("the provided old value has to be a pointer")
```

Comment:
why changing this from `fmt.Errorf` to `errors.New`?  
i like having a consistent `fmt` usage. Is there a benefit despite a little performance advantage, which should be neglectable as the error should not happen in the first place? :D

Reply:
No, it's only about using the right tool for the right purpose (and it's a little bit faster)

Reply:
for me, consistency would be the winner here.. also one less import..  
but also no hard feelings about it, if you want to keep it like this..